lib/src/facts/resolver.cc

```diff
@@ -24,6 +26,23 @@ namespace facter { namespace facts {
                 throw invalid_name_pattern_exception(ex.what());
             }
         }
+
+#ifdef LEATHERMAN_USE_LOCALES
+        // build Accept-Language list for HTTP-based resolvers
+        const auto& loc = leatherman::locale::get_locale();
+        if (std::has_facet<boost::locale::info>(loc)) {
+            const auto& info = std::use_facet<boost::locale::info>(loc);
+            string lang = info.language();
+            // use country code when available, but add fallback to base lang
+            if (!info.country().empty())
+                lang += "-" + info.country() + ", " + info.language();
+            // always include English (en) as a fallback
+            if (info.language() != "en")
+                lang += ", en";
```

Comment:
If my reading of the HTTP spec (https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4) is correct, Accept-Language does not include the character encoding. Correct?